tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1753,11 +1753,29 @@ def window(self, size, shift=None, stride=1, drop_remainder=False):
 
     A "window" is a finite dataset of flat elements of size `size` (or possibly
     fewer if there are not enough input elements to fill the window and
-    `drop_remainder` evaluates to false).
+    `drop_remainder` evaluates to `False`).
+
+    The `shift` argument determines the number of input elements by which
+    the window moves on each iteration.  The first element in the `k`th window
+    will be element
+
+    ```
+    1 + (k-1) * shift
```

Comment:
Can we write this to be zero-indexed? Then the formula simplifies to `k * shift`

Reply:
Changed the equations in this sentence so that windows and elements are numbered starting at zero.